src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1832,14 +1832,6 @@ private bool ExpressionMatchExactly(BoundExpression node, TypeSymbol t, ref Hash
 
             if (node.Kind == BoundKind.TupleLiteral)
             {
-                if ((object)node.Type != null)
-                {
-                    // expression has a natural type and 
```

Comment:
This might need a comment why we can still find a suitable exact match even after identity conversion with existing type failed.
The part that we may be targeting not a canonical tuple type may not be obvious to someone who is new to this code.